test/utils/noderesourcetopologies/noderesourcetopologies.go

```diff
@@ -186,16 +186,17 @@ func FindFromList(nrts []nrtv1alpha1.NodeResourceTopology, name string) (*nrtv1a
 	return nil, fmt.Errorf("failed to find NRT for %q", name)
 }
 
-func findZoneByName(nrt nrtv1alpha1.NodeResourceTopology, zoneName string) (*nrtv1alpha1.Zone, error) {
-	for idx := 0; idx < len(nrt.Zones); idx++ {
-		if nrt.Zones[idx].Name == zoneName {
-			return &nrt.Zones[idx], nil
-		}
+// AvailableFromZone returns a ResourceList of all available resources under the zone
+func AvailableFromZone(z nrtv1alpha1.Zone) corev1.ResourceList {
```

Comment:
this is fine, but we should (briefly) expain "why/how" in the commit message rather than "what", because "what" must be evident from the PR itself, and if the PR is too big to make it evident, we will need to split it.